.github/workflows/ci.yml

```diff
@@ -0,0 +1,57 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        node: [12.x, 14.x, 15.x]
```

Comment:
```suggestion
        node: [12.x, 14.x, 15.x]
        architecture: [x64]
        include:
          - os: windows-2016
            node_version: 12
            architecture: x86
```

Reply:
What does this do and why does it improve the CI?